### PR TITLE
Gordon pr1930 tweek

### PR DESCRIPTION
**Pull request by hutchig**
The PR is built on top of pull 1930 and tries to take the very valid points from that forwards.

### CODE REVIEW

File: tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/clientserver/Bulkhead5RapidRetry12MethodSynchBean.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+/**
+ * A simple method level synchronous @Bulkhead bean that has a retry option, 
+ * with a waitingTaskQueue value that should be ignored.
+ * 
+ * @author Andrew Pielage
+ */
+public class Bulkhead5RapidRetry12MethodSynchBean implements BulkheadTestBackend {
+    
+    @Override
+    @ApplicationScoped
```
Comment (Emily-Jiang) [inline tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/clientserver/Bulkhead5RapidRetry12MethodSynchBean.java]: This is not a producer. Why put ApplicationScoped annotation on?

Comment (hutchig) [reply tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/clientserver/Bulkhead5RapidRetry12MethodSynchBean.java]: This is not 'new' code. I just renamed it to a more consistent name. The author tag says Andrew Pielage.  If the original ApplicationScvoped came from me IMHO I think it is a good thing..to have some variety and more would be better. We don't have great coverage of scopes (standard or custom) and too many of the test beans have the same scope. For example - do we test that the bulkhead 'in' count is per target bean instance correctly scoped and not per bean class or call site...no, not yet.

Comment (Emily-Jiang) [reply tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/clientserver/Bulkhead5RapidRetry12MethodSynchBean.java]: It is wrong to have that annotation. I

Comment (hutchig) [reply tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/clientserver/Bulkhead5RapidRetry12MethodSynchBean.java]: Why not? Can't one have fault tolerance to a AppScoped singleton? Please explain? Sure the test does not exploit it but it is a valid thing to have and thus test. Please explain why not.

**Comment by Emily-Jiang** [comment]
@ApplicationScoped is only used for controlling bean's lifecycle. In this case, the annotation is directly specified on a non-producer method, which does not produce a bean. It is wrong to be there.

File: tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/clientserver/Bulkhead5RapidRetry12MethodSynchBean.java (hunk 1 above)
Comment (hutchig) [reply tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/clientserver/Bulkhead5RapidRetry12MethodSynchBean.java]: Agh - just spotted it was on the method and not the class and hence a scoping  - apologies. 
 I can only assume that this is a copy and paste error either by Andrew or me. I will remove it with an additional commit tomorrow first thing.

**Comment by hutchig** [comment]
Commit 912cb20 added to the PR removes the method level @ApplicationScoped annotations.

**Comment by Emily-Jiang** [comment]
Thanks @hutchig @Pandrex247 !

**Pull request merged**